cmake/SetBuildOptions.cmake

```diff
@@ -35,7 +35,13 @@ elseif(COMPILER STREQUAL "Intel" OR COMPILER STREQUAL "IntelLLVM")
     set(CMAKE_C_FLAGS_DEBUG "-O0 -g")
     set(CMAKE_C_FLAGS_RELEASE "-O2 -debug minimal")
     set(CMAKE_Fortran_FLAGS "-free -qno-opt-dynamic-align -ftz -traceback -convert big_endian -assume byterecl -assume realloc_lhs -fp-model source -qopenmp")
-    set(CMAKE_Fortran_FLAGS_DEBUG "-O0 -g -check uninit -check bounds -check pointers -fpe0 -check noarg_temp_created")
+    if(USE_PDAF)
```

Comment:
Let's avoid model-specific logic here; compiler and linker flags that pertain to program correctness should be model-independent.

Reply:
I agree. I am currently trying with `-check all` instead of the individual check flags. Could this become the model-independent default?

Thanks a lot for the porting-guide, I will have a look at this and see what I can also apply to PDAF-flags to have flags compatible.

Reply:
> Could this become the model-independent default?

Yep; `USE_PDAF` can be removed here.